vueapp/src/views/TilePad.vue

```diff
@@ -140,6 +174,26 @@ export default {
 			let textToSpeak = this.sentenceTiles.map(tile => tile.text).join(' ');
 			this.speakText(textToSpeak);
 			this.logTaps();
+		}, 
+		saveSentence () {
+			var savedSentences =TileData.sentences.tileData;
+			let textToSave = this.sentenceTiles.map(tile => tile.name).join(' ');
+			let images = this.sentenceTiles.map(tile => tile.image);
+			var last_id = (savedSentences).length;
+			var sentence = { 'name':textToSave,
+				'text' :textToSave,
+				'id': last_id, 
+				'accent':'mint',
+				'image': images };
+			var myJSON = JSON.stringify(sentence);
+			console.log(myJSON);
+			savedSentences[last_id]= sentence;
+			console.log(TileData.sentences.tileData);
+			TileData.sentences.tileData[last_id]= sentence;
```

Comment:
I think this may be causing some issues as well. We should be using the Vuex store to save this data. Probably best to be out of TileData and it's own state object as well. If you would like I can add this part of it or share some documentation/point to other locations in the code that does this.. 

Also JSON.stringify should not be needed.

Reply:
Sure! I only used the logs to make sure the correct data were being saved. I removed them and it works fine!